src/NUnitFramework/framework/Is.cs

```diff
@@ -534,15 +534,29 @@ public static SamePathOrUnderConstraint SamePathOrUnder(string expected)
         /// Returns a constraint that tests whether the actual value falls
         /// inclusively within a specified range.
         /// </summary>
-        /// <remarks>from must be less than or equal to true</remarks> 
-        /// <param name="from">Inclusive beginning of the range. Must be less than or equal to to.</param>
-        /// <param name="to">Inclusive end of the range. Must be greater than or equal to from.</param>
+        /// <remarks>The <paramref name="from"/> value must be less than or equal to the <paramref name="to"/> value.</remarks> 
+        /// <param name="from">Must be less than or equal to the <paramref name="to"/> value.</param>
+        /// <param name="to">Must be greater than or equal to the <paramref name="from"/> value.</param>
         /// <returns></returns>
         public static RangeConstraint InRange(IComparable from, IComparable to)
```

Comment:
Use of IComparable was an old old experiment at making constraints more type-safe. Obviously, this issue shows that the approach doesn't work when we allow arbitrary comparers to be used. So I'd go back to two-object constructor and syntax elements in all cases.